app/views/proposal/index.html.haml

```diff
@@ -76,7 +76,7 @@
                 - progress_percentage = event.calculate_progress
                 .progress
                   %div{class: "progress-bar #{event_progress_color(progress_percentage)}", style: "width:#{progress_percentage}%;"}
-                    = event.progress_status.reject{ |_key, value| value }.length
+                    = event.progress_status.reject{ |_key, value| value or value.nil? }.length
```

Comment:
`|_key, value| value || value.nil?`

http://phrogz.net/programmingruby/language.html#table_18.4

(However, rubocop should complain about this ...)